vtex/faststore
# fix(api): Adding fields name, level and parentId in collection schema

## What's the purpose of this pull request?

This PR adds three fields in the 'allCollections' query: name, level and parentId. I need them to build the menu inside my store header, but they were not in the Gatsby data layer. In front, I use this data to build the category tree for the menu.

## How it works? 

Adds name, level and parentId properties in the GraphQL schema;
Adds level and parentId properties in CategoryTree interface;
Updates collection fetching in query resolver to append parentId.

## How to test it?

Check the base.store PR created below and try to run allCollection and collection queries with the new fields.

### `base.store` Deploy Preview

[vtex-sites/base.store#223](https://github.com/vtex-sites/base.store/pull/223)



## References
Followed the tutorial from Office Hours of 06/Jan/2022.
